faces/3.0/_index.md

```diff
@@ -1,25 +1,28 @@
 ---
-title: "Jakarta Server Faces 3.0 (milestone)"
-date: 2020-06-12
+title: "Jakarta Server Faces 3.0"
+date: 2020-09-23
 summary: "Release for Jakarta EE 9"
 ---
 Jakarta Server Faces defines an MVC framework for building user interfaces for web applications,
 including UI components, state management, event handing, input validation, page navigation, and
 support for internationalization and accessibility.
 
-* [Jakarta Server Faces 3.0 Release Record](https://projects.eclipse.org/projects/ee4j.faces/releases/3.0.0)
+* [Jakarta Server Faces 3.0 Release Record](https://projects.eclipse.org/projects/ee4j.faces/releases/3.0.0/plan)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Server Faces 3.0 Specification Document](./faces-3.0-SNAPSHOT.pdf) (PDF)
-* [Jakarta Server Faces 3.0 Specification Document](./faces-3.0-SNAPSHOT.html) (HTML)
+* [Jakarta Server Faces 3.0 Specification Document](./jakarta-faces-3.0.pdf) (PDF)
+* [Jakarta Server Faces 3.0 Specification Document](./jakarta-faces-3.0.html) (HTML)
 * [Jakarta Server Faces 3.0 Javadoc](./apidocs)
-* [Jakarta Server Faces 3.0 TCK]()([sig](),[sha](),[pub]())
+* [Jakarta Server Faces 3.0 Jsdoc](./jsdoc)
+* [Jakarta Server Faces 3.0 Renderkitdoc](./renderkitdoc)
+* [Jakarta Server Faces 3.0 VDLDoc](./vdldoc)
+* [Jakarta Server Faces 3.0 TCK](https://download.eclipse.org/jakartaee/faces/3.0/jakarta-faces-tck-3.0.0.zip)([sig](https://download.eclipse.org/jakartaee/faces/3.0/jakarta-faces-tck-3.0.0.zip.sig),[sha](https://download.eclipse.org/jakartaee/faces/3.0/jakarta-faces-tck-3.0.0.zip.sha256),[pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
 * Maven coordinates
-  * [jakarta.faces:jakarta.faces-api:jar:3.0.0-RC1](https://search.maven.org/artifact/jakarta.faces/jakarta.faces-api/3.0.0-RC1/jar)
+  * [jakarta.faces:jakarta.faces-api:jar:3.0.0](https://search.maven.org/artifact/jakarta.faces/jakarta.faces-api/3.0.0/jar)
 
 
 # Compatible Implementations
 
-* [Mojarra 3.0.0-RC2](https://eclipse-ee4j.github.io/mojarra)
+* [Mojarra 3.0.0-RC4](https://eclipse-ee4j.github.io/mojarra)
```

Comment:
The referenced content via this link has not been updated for Mojarra 3.0.0.  It's still referencing 2.3 and it's dependencies.  Where can this Mojarra 3.0.0-RC4 be accessed?  I found this tag (https://github.com/eclipse-ee4j/mojarra/releases/tag/3.0.0-RC4-RELEASE), but there is no binary associated with it that I can find.  It looks like the CR and TCK Results reference this same page. I won't hold up my vote, but this has to be resolved before this PR can be merged.

Reply:
RC4 is primarily available here: 

https://jakarta.oss.sonatype.org/content/repositories/staging/org/glassfish/jakarta.faces/3.0.0-RC4/

But I've just added the binaries to the tag: https://github.com/eclipse-ee4j/mojarra/releases/tag/3.0.0-RC4-RELEASE

I'll update them to the link to the binaries.